examples/03-Running-on-multiple-GPUs-or-on-CPU.ipynb

```diff
@@ -0,0 +1,742 @@
+{
```

Comment:
<p>Similar to my comment above. </p><p> </p><p>Can we use write the section in a way, how we would use NVTabular on CPU without using GPU commands before?</p><p>This would imply that we run nvt.Dataset() on GPU and then move to cpu. We should use directly</p><p> </p><pre class="ql-syntax" spellcheck="false">nvt.Dataset(..., cpu=True)
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
good point! made the correction